packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -105,6 +106,8 @@ export const availableCustomTypesReducer: Reducer<
       };
     }
     case getType(createCustomTypeCreator.success): {
+      action.payload.newCustomType.__status = CustomTypeStatus.New;
```

Comment:
better to set the `__status` in the saga before the `createCustomTypeCreator.success` action is dispatched. Ideally in the `createCustomType` method